tasklist/els-schema/src/main/java/io/camunda/tasklist/util/IndexSchemaValidatorUtil.java

```diff
@@ -76,9 +76,11 @@ public Map<String, IndexMapping> filterIndexMappings(
       final Map<String, IndexMapping> indexMappings, final IndexDescriptor indexDescriptor) {
     return Maps.filterEntries(
         indexMappings,
-        e -> e.getKey().matches(indexDescriptor.getAllVersionsIndexNameRegexPattern()));
+        e -> e.getKey().matches(indexDescriptor.getAllVersionsIndexNameRegexPattern()) &&
+            "true".equals(e.getValue().getDynamic())); // filter out dynamic mappings - not supported by schema migration
```

Comment:
Let's put the "expensive" regex matching at the second position :) 
```suggestion
        e ->  "true".equals(e.getValue().getDynamic())) && // filter out dynamic mappings - not supported by schema migration 
                 e.getKey().matches(indexDescriptor.getAllVersionsIndexNameRegexPattern());
```